controllers/kogitoserverlessworkflow_controller.go

```diff
@@ -61,181 +60,43 @@ type KogitoServerlessWorkflowReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.2/pkg/reconcile
 func (r *KogitoServerlessWorkflowReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log := ctrllog.FromContext(ctx)
+	logger := ctrllog.FromContext(ctx)
 
 	// Make sure the operator is allowed to act on namespace
 	if ok, err := platform.IsOperatorAllowedOnNamespace(ctx, r.Client, req.Namespace); err != nil {
 		return reconcile.Result{}, err
 	} else if !ok {
-		log.Info(fmt.Sprintf("Ignoring request because the operator hasn't got the permissions to work on namespace %s", req.Namespace))
+		logger.Info(fmt.Sprintf("Ignoring request because the operator hasn't got the permissions to work on namespace %s", req.Namespace))
 		return reconcile.Result{}, nil
 	}
 
 	// Fetch the Workflow instance
-	workflow := &apiv08.KogitoServerlessWorkflow{}
+	workflow := &operatorapi.KogitoServerlessWorkflow{}
 	err := r.Client.Get(ctx, req.NamespacedName, workflow)
 	if err != nil {
 		if errors.IsNotFound(err) {
 			return ctrl.Result{}, nil
 		}
-		log.Error(err, "Failed to get KogitoServerlessWorkflow")
+		logger.Error(err, "Failed to get KogitoServerlessWorkflow")
 		return ctrl.Result{}, err
 	}
 
 	// Only process resources assigned to the operator
 	if !platform.IsOperatorHandlerConsideringLock(ctx, r.Client, req.Namespace, workflow) {
-		log.Info("Ignoring request because resource is not assigned to current operator")
+		logger.Info("Ignoring request because resource is not assigned to current operator")
 		return reconcile.Result{}, nil
 	}
 
-	switch workflow.Status.Condition {
-
-	// If the status condition is None or Waiting for platform let's try to start a build!
-	case apiv08.NoneConditionType, apiv08.WaitingForPlatformConditionType:
```

Comment:
is safe to remove these reconcile steps?

Reply:
Yes, we refactored the whole reconciliation steps. See the `reconciler_prod.go` file. This code was migrated there.